.github/workflows/reusable_checks.yml

```diff
@@ -52,6 +52,14 @@ jobs:
     - name: Check Python formatting
       run: cmake --build build --target black-format-check
 
+    - name: Run check-license
+      run: |
+        ./check_license/check_headers.sh . "Apache-2.0 WITH LLVM-exception" -v
```

Comment:
heh, not sure how I missed that - script (and dir) should not be in the root of the repo